src/components/classification/classification.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { generateAnchorAttrsFromLink, getCustomAttributes, htmlAttributes } from "@/framework/widgets/attributes";
+import { WidgetContext } from "@/framework/widgets/widget-context";
+import { classNames } from "@/framework/utils/classNames";
+import { LinkModel } from "@/framework/interfaces/LinkModel";
+import { StyleGenerator } from "../styling/style-generator.service";
+import { OffsetStyle } from "../styling/offset-style";
+import { Alignment } from "../styling/alignment";
+import { ButtonType } from "../styling/button-types";
+import { ClassificationRestService } from './classification.service'
+import dompurify from "isomorphic-dompurify";
+
+const mapTaxonProperties = (taxon: any, taxonomyName: string, viewUrl?: string) =>{
+    const children = [];
+
+    taxon.SubTaxa.forEach((t) =>{
+        child.SubTaxa = mapTaxonProperties(child, taxonomyName);
+        child.UrlName = getTaxaUrl(taxonomyName, child.UrlName, viewUrl);
+        children.push(child);
+    });
+
+    return children;
+}
+
+const getTaxaUrl = (taxonomyName: string, taxonUrl: string, viewUrl?: string) => {
+    if (viewUrl === null)
+    {
+        return "#";
+    }
+    let queryString = ''
+    // we need to think how to add the query
+    // IQueryCollection queryCollection = this.httpContextAccessor.HttpContext.Request.Query;
+    // if (queryCollection != null && queryCollection.Count > 0)
+    // {
+    //     var whitelistedQueryParams = new[] { QueryParams.Action, QueryParamNames.Site, QueryParamNames.Provider, QueryParamNames.Culture };
+    //     var filteredQueryCollection = queryCollection?.Where(x => whitelistedQueryParams.Contains(x.Key));
+    //     queryString = QueryString.Create(filteredQueryCollection).ToString();
+    // }
+
+    if (taxonUrl.startsWith('/'))
+    {
+        taxonUrl = taxonUrl.substring(1);
+    }
+
+     return `${viewUrl}/-in-${taxonomyName},${taxonUrl.replaceAll("/", ",")}` + queryString;
+}
+
+export async function Classification(props: WidgetContext<ClassificationEntity>) {

```

Comment:
The classifcation widget should  filter the items in the content list widget. Is this handled ?